entrypoint.sh

```diff
@@ -1,42 +1,48 @@
 #!/bin/bash
 
```

Comment:
How about adding a comment here with an example of how you would call this script?